%YAML:1.0
---
calibrationTimeChar: "Wed Jan 15 11:00:19 2025\n"
calibrationTime_t: 1736910019
imageWidth: 2240
imageHeight: 1740
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.1630491208257970e+03, 0., 1.1052647657768491e+03, 0.,
       1.1620851806820926e+03, 8.5261105409059041e+02, 0., 0., 1. ]
distCoeffs: !!opencv-matrix
   rows: 1
   cols: 4
   dt: d
   data: [ -9.0129642121882458e-02, 1.0019545125818970e-01,
       -2.3028608185322104e-04, -1.5148680171757158e-04 ]
newCameraMatrixAlpha1: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.1355098231893537e+03, 0., 1.1042540796843864e+03, 0.,
       1.1305135017773225e+03, 8.5203736390904726e+02, 0., 0., 1. ]
newDisCoffesAlpha1: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 1.0228392871092135e-02, -3.0181748650396543e-02,
       1.2258404824329333e-04, 2.9238174834256834e-04,
       2.4399511344129602e-02 ]
newCameraMatrixAlpha0: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.1476853866937747e+03, 0., 1.1038321044767658e+03, 0.,
       1.1467912492895020e+03, 8.5124873208624012e+02, 0., 0., 1. ]
newDistCoeffsAlpha0: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 7.7103343247299557e-03, -2.6069528540477480e-02,
       2.7289445426949013e-05, 1.1969971327340417e-04,
       2.3176401089065182e-02 ]
